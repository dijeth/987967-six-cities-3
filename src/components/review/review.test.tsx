import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Review from './review';

const mock = {
  id: `id-1`,
  userName: `userName-1`,
  userPicture: `userPicture-1`,
  rating: 4.8,
  description: `description-1`,
  date: new Date(1582194976548).toISOString()
};

it(`<Review /> should be render correctly`, () => {
  const review = renderer.create(<Review {...mock} />).toJSON();
  expect(review).toMatchSnapshot();
});
