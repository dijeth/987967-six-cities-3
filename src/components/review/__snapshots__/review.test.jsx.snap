// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Review /> should be render correctly 1`] = `
Array [
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="userPicture-1"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      userName-1
    </span>
  </div>,
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "100%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      description-1
    </p>
    <time
      className="reviews__time"
      dateTime="2020-02-20"
    >
      February 2020
    </time>
  </div>,
]
`;
